src/main/kotlin/io/csbroker/apiserver/common/interceptor/ratelimit/RateLimiter.kt

```diff
@@ -0,0 +1,5 @@
+package io.csbroker.apiserver.common.interceptor.ratelimit
+
+interface RateLimiter {
```

Comment:
차후 RateLimiter에 다른 Abstract Function이 추가될 일이 있나요? 
없으면 SAM이라는걸 명시해주는 게 좋을 것 같구 확장가능성이 있다면 지금 이대로가 좋은 것 같습니다~
```suggestion
fun interface RateLimiter {
```

Reply:
뭔가.. ip대역 제한 말고, user에 따른 제한이라던가.. 같은 걸 추후 고려해볼 수 있지 않을까 싶어서 그래놨습니당.